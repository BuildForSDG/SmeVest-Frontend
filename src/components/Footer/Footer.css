.Footer {
  width: 100%;
  min-height: 40vh;
  background: #011a29;
  color: #ddd;
  padding: 7rem 0 1rem;
  overflow: hidden;
  font-family: 'Karla', sans-serif;
}

.Footer a {
  color: #ddd;
  text-decoration: none;
  transition: all 300ms linear;
}

.Footer a:hover {
  color: #1fc2d8;
}

.Footer .Logo {
  display: flex;
  justify-content: flex-start;
  align-items: center;
}

.Footer .Logo .logo-img {
  margin: 0;
  padding: 0;
}

.Footer .Address {
  margin: 0 0.5rem;
  display: flex;
  flex-direction: column;
  justify-content: center;
  align-items: flex-start;
}

.Footer .Address-Text {
  display: flex;
  align-items: center;
  line-height: 1;
  font-size: 0.9rem;
  margin-top: 0.4rem;
}

.Footer .Address .first {
  margin-top: 0.8rem;
}

.Footer .Address-Icon {
  margin-right: 1rem;
}

.Footer .Links {
  display: flex;
  justify-content: flex-start;
  align-items: center;
}

.Footer .Quicklinks {
  list-style: none;
  align-self: center;
  padding: 0;
  margin: 0 2.5rem 0 0;
}

.Footer .Quicklinks li {
  margin: 0.8rem 0;
}

.Footer .Quicklinks .Link {
  font-size: 0.9rem;
}

.Footer .Icons {
  display: flex;
  align-items: center;
}

.Footer .SocialIcons {
  display: flex;
}

.Footer .SocialIcons a {
  display: inline-block;
  margin-right: 1rem;
}

.Footer .SocialIcons .Icon {
  border: 1px solid #ccc;
  border-radius: 50%;
  padding: 0.4rem;
  transition: all 300ms ease;
  font-size: 2.3rem;
}

.Footer .SocialIcons .Icon:hover {
  background: #1fc2d8;
  color: #fff;
  border-color: #1fc2d8;
}

.Footer .Copyright {
  font-size: 0.9rem;
  text-align: center;
}

@media screen and (max-width: 600px) {
  .Footer .Row {
    display: flex;
    flex-direction: column;
  }

  .Footer .Logo {
    justify-content: center;
    margin-bottom: 1rem;
  }

  .Footer .Address {
    justify-content: flex-start;
    align-items: center;
    margin-bottom: 1rem;
  }

  .Footer .Address-Text {
    justify-content: start;
    align-items: start;
    line-height: 1.5;
    font-size: 0.9rem;
    text-align: center;
  }

  .Footer .Links {
    width: 100%;
    flex-direction: column;
    justify-content: center;
    margin-bottom: 1rem;
  }

  .Footer .Quicklinks {
    text-align: center;
  }

  .Footer .SocialIcons {
    width: 100%;
    display: flex;
    justify-content: center;
    padding: 0;
  }

  .Footer .Copyright {
    margin: 1rem 0;
  }
}
