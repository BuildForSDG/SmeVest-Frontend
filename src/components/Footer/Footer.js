import React from 'react';
import { Link } from 'react-router-dom';
import { Container, Row, Col } from 'react-bootstrap';
import logo from '../../assets/images/logo.png';
import { Room, Phone, Mail, Facebook, Twitter, Instagram, LinkedIn } from '@material-ui/icons';
import './Footer.css';

const Footer = () => (
  <div className="Footer">
    <Container>
      <Row>
        <Col sm className="Logo">
          <Link to="/">
            <img src={logo} alt="Logo" />
          </Link>
        </Col>
        <Col sm className="Address">
          <p>
            <Room /> No. 3 Andela Estate, NY.
          </p>
          <p>
            <Phone /> +234 (80) 1234 5678
          </p>
          <p>
            <Mail /> <a href="mailto:info@smevest.org">info@smevest.org</a>
          </p>
        </Col>
        <Col sm className="Links">
          <ul className="Quicklinks">
            <li>
              <Link to="/about-use">About Us</Link>
            </li>
            <li>
              <Link to="/wwd">What We Do</Link>
            </li>
            <li>
              <Link to="/faq">FAQ</Link>
            </li>
          </ul>
          <ul className="Quicklinks">
            <li>
              <Link to="/career">Career</Link>
            </li>
            <li>
              <Link to="/blog">Blog</Link>
            </li>
            <li>
              <Link to="/contact-us">Contact Us</Link>
            </li>
          </ul>
        </Col>
        <Col sm className="Icons">
          <ul className="SocialIcons">
            <a href="https://twitter.com/yasholma" rel="noopener noreferrer" target="_blank">
              <Facebook />
            </a>
            <a href="https://twitter.com/yasholma" rel="noopener noreferrer" target="_blank">
              <Twitter />
            </a>
            <a href="https://twitter.com/yasholma" rel="noopener noreferrer" target="_blank">
              <Instagram />
            </a>
            <a href="https://twitter.com/yasholma" rel="noopener noreferrer" target="_blank">
              <LinkedIn />
            </a>
          </ul>
        </Col>
      </Row>
    </Container>
    <p className="Copyright">&copy; Copyright 2020 SmeVest. All rights reserved. </p>
  </div>
);

export default Footer;
