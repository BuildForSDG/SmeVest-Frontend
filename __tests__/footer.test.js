import React from 'react';
import { shallow } from 'enzyme';
import Footer from '../src/components/Footer/Footer.js';

describe('<Footer /> Component', () => {
  test('should render footer component', () => {
    expect(shallow(<Footer />)).toMatchSnapshot();
  });
});
