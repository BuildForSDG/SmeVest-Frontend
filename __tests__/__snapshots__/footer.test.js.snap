// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> Component should render footer component 1`] = `ShallowWrapper {}`;
